### PR TITLE
Implement get_expired stats

### PR DESCRIPTION
**Pull request by sergiocarlosmorales**
From issue #126, https://code.google.com/p/memcached/issues/detail?id=289:
Implement get_expired

### CODE REVIEW

File: t/stats.t [hunk 1]
```diff
@@ -203,6 +205,15 @@ is(0, $stats->{'evictions'});
 is(0, $stats->{'reclaimed'});
 is(0, $stats->{'lrutail_reflocked'});
 
+# item expired
+print $sock "set should_expire 0 1 6\r\nfooval\r\n";
+is(scalar <$sock>, "STORED\r\n", "set item to expire");
+sleep(2);
```
Comment (sergiocarlosmorales) [inline t/stats.t]: if there is a better way to do this, let me know

Comment (dormando) [inline t/stats.t]: you can set an expiration time of 31 days. Past 30 days it's interpreted as a unix timestamp, so any still relatively-small value will be immediately expired.

Comment (sergiocarlosmorales) [inline t/stats.t]: thanks, forgot about that, fixed.

**Comment by dormando** [comment]
thanks! So sorry for the massive delay. This has been pushed to the 'next' branch and will go out in the next release.

**Pull request closed without merging**